sql/src/test/java/io/crate/planner/SubQueryPlannerTest.java

```diff
@@ -198,25 +198,24 @@ public void testJoinOnSubSelectsWithLimitAndOffset() throws Exception {
         assertThat("1 node, otherwise mergePhases would be required", left.nodeIds().size(), is(1));
         assertThat(left.orderBy(), isSQL("OrderByPositions{indices=[1], reverseFlags=[false], nullsFirst=[null]}"));
         assertThat(left.collectPhase().projections(), contains(
-            isTopN(5, 2)
+            isTopN(10, 2)
         ));
         Collect right = (Collect) nl.right();
         assertThat("1 node, otherwise mergePhases would be required", right.nodeIds().size(), is(1));
         assertThat(((RoutedCollectPhase) right.collectPhase()).orderBy(), isSQL("doc.t2.b"));
         assertThat(right.collectPhase().projections(), contains(
-            isTopN(10, 5),
+            isTopN(5, 5),
```

Comment:
why is changing this complete test needed ?

Reply:
Because the limit  Left 5, Right 10,  (right bigger) results in a table switch